### ISSUE TITLE
Remove a space in CC license text and display a URI to the CC license text 

### ISSUE THREAD

**Issue by sstrecker** [sstrecker/emisa-latex-package#27]
There seems to be an extra space before the word 'license' at the end of the \CCBYNCSAFour command output (see attached output).

Should the URI to the license text be displayed as well (i.e. https://creativecommons.org/licenses/by-sa/4.0/)?

![cclicense](https://cloud.githubusercontent.com/assets/5647896/13225882/31643b66-d98f-11e5-9964-cf07b0bc5f4b.jpg)

**Comment by sieversMartin** [comment]
I just take the macro `\doclicenseLongText` from the `doclicense` package.

``` latex
\newcommand{\doclicenseLongText}{%
  \doclicense@lang@thisDoc\space
  \href{\doclicenseURL}{\doclicenseLongType\space\enquote{\doclicense@longName}}
  \doclicense@lang@word@license.\xspace%
}
```

**Comment by sieversMartin** [comment]
Fix by:

``` latex
\renewcommand{\doclicenseLongText}{%
   \doclicense@lang@thisDoc\space
   \href{\doclicenseURL}{\doclicenseLongType\space\enquote{\doclicense@longName}}%
   \doclicense@lang@word@license.\xspace%
}
```

**Comment by sieversMartin** [comment]
I changed the output to use the macro definition directly. I as well sent a bug report to the author of the `doclicense` package.

**Comment by sieversMartin** [comment]
The license text has always been a hyperlink, so the URL is at hand. However, I changed the method to link to the correct website now.

IMHO that is sufficient.

**Comment by sieversMartin** [comment]
Fixed in c2cd00d3011c998ced454b9d048af1daa1a69046
